crates/bdk/tests/wallet.rs

```diff
@@ -809,7 +813,10 @@ fn test_create_tx_add_utxo() {
         )
         .unwrap();
 
-    let addr = Address::from_str("2N1Ffz3WaNzbeLFBb51xyFMHYSEUXcbiSoX").unwrap();
+    let addr = Address::from_str("2N1Ffz3WaNzbeLFBb51xyFMHYSEUXcbiSoX")
+        .unwrap()
+        .require_network(Network::Testnet)
+        .unwrap();
```

Comment:
In test code, if you want, you can just call `assumed_checked` instead of `require_network`.